chapters/dae.tex

```diff
@@ -102,10 +102,11 @@ \chapter{Modelica DAE Representation}\label{modelica-dae-representation}
 \end{example}
 
 The generated set of equations is used for simulation and other analysis activities.
-Simulation means that an initial value problem is solved, i.e., initial values have to be provided for the states $x$, \cref{initialization-initial-equation-and-initial-algorithm}.
+In this specification simulation means that first that initial values are found for the states $x$, \cref{initialization-initial-equation-and-initial-algorithm}.
```

Comment:
This sentence looks incomplete and has a typo.  Would this work better, assuming that we actually have a proper definition of _simulation_ somewhere else?
```suggestion
Here, \emph{simulation} includes solving the initialization problem where -- among other -- initial values for the states $x$ are found, see \cref{initialization-initial-equation-and-initial-algorithm}.
```